can/broadcastmanager.py

```diff
@@ -34,28 +35,42 @@ class CyclicSendTaskABC(CyclicTask):
     Message send task with defined period
     """
 
-    def __init__(self, message, period):
+    def __init__(self, messages, period):
         """
-        :param can.Message message: The message to be sent periodically.
-        :param float period: The rate in seconds at which to send the message.
+        :param List[can.Message] messages: The message to be sent periodically.
```

Comment:
This should probably be something like `Union[List[can.Message], can.Message]`, and the docstring should be updated.